Milestone 1: Add index.html, style.css, and images for the site

indigodavid opened the pull request:
This site shows a first draft of the Youtube Player Page with its
links, comments sections and related sections.

It was added:

- index.html
- style.css
- images in PNG and SVG

indigodavid committed:
Add index.html, style.css, and images for the site

This site shows a first draft of the Youtube Player Page with its
links, comments sections and related sections.

indigodavid commented on index.html:
```diff
@@ -0,0 +1,157 @@
+                        <div><img src="" alt="Video Thumbnail"></div>
+                        <div>
+                            <h3 class="title">Lorem ipsum dolor sit.</h3>
+                            <p for="" class="author">Author 2</p>
```
The "for" attribute is not correct for a p tag. It would be better to remove this attribute from all the p elements in the "related-videos " section.

indigodavid commented on index.html:
```diff
@@ -0,0 +1,157 @@
+            <section id="related-videos">
+                <ul>
+                    <li class="related-link">
+                        <div><img src="" alt="Video Thumbnail"></div>
```
You should consider adding placeholder images to this section so you can check their behavior on the website.

indigodavid commented on style.css:
```diff
@@ -0,0 +1,195 @@
+    padding: 0;
+}
+
+#search input, button{
```
If you need this elements to have similar formatting it would be better to create a separate class with the general format and use the individuals classes for the specifics. You may also need to edit the html accordingly.

indigodavid reviewed (commented):
# Changes required ♻️ 

Your website does not comply with the linters yet. It is required, to comply with Microverse standards, for you to edit and pass the Linters modifications. Most of your Linters errors are related to indentation, but be sure to check them thoroughly.

Also please check the comments below to make minor changes to improve your site.

Keep on coding! 👍

m05e5 reviewed (approved):
# Hi @indigodavid , 

### Status: Approved ✔️
![goodjob](https://user-images.githubusercontent.com/60631456/130260474-3c2d5e5c-96df-43de-a0db-8d0059c10862.gif)


.

Your project is complete!  There are some optional changes I've suggested to you to make your project best, other than that nothing else to say.
Congratulations! 🎉 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

[Highly Recommended]
- _( optional )_ Please next time try to be more gentle in your review. Some developers feel stressed so a gentle review will help them feel better.

_Apart from that you really did a great job_ 

**You may close this Pull Request and merge the changes**

  
**Keep being awesome devs and good luck with your next milestone!**
-----

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_m05e5_**

------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

indigodavid committed:
Modify files to comply review

The files index.html and style.css have been modified so they comply
with Linters. Add images as placeholders for users and thumbnails.

indigodavid committed:
mend

indigodavid committed:
mend
